### PR TITLE
Add NotSupportedError for unknown ICE server schema

### LINKED ISSUE #1346: Ignore unsupported url types in setConfiguration
From EKR: https://lists.w3.org/Archives/Public/public-webrtc/2017May/0166.html

(setConfiguration)

```
3. For each url in server.urls parse url and obtain scheme
name. If the scheme name is not implemented by the browser, or if
parsing based on the syntax defined in [ RFC7064] and [RFC7065]
fails, throw a SyntaxError.
```

This seems unwise as it makes it very hard to introduce new schemes.
Why aren't they just ignored?

See https://github.com/w3c/webrtc-pc/issues/1278 for context.

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #1346

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/webrtc-pc/issue#1346.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/webrtc-pc/c602d52...ffcb11b.html)

### CODE REVIEW

**Comment by fluffy** [comment]
@fippo Does this PR look right to you ?

File: webrtc.html [hunk 1]
```diff
@@ -2793,12 +2793,25 @@
...
                       <li>
                         <p>For each <var>url</var> in
-                        <code><var>server</var>.urls</code> parse
-                        <var>url</var> and obtain <var>scheme name</var>. If
-                        the <var>scheme name</var> is not implemented by the
-                        browser, or if parsing based on the syntax defined in
-                        [[!RFC7064]] and [[!RFC7065]] fails, <a>throw</a> a
-                        <code>SyntaxError</code>.</p>
+                        <code><var>server</var>.urls</code> parse the
+                        <var>url</var> using the genric URI syntax
```
Comment (fippo) [inline webrtc.html]: genric -> generic

File: webrtc.html [hunk 2]
```diff
@@ -2793,12 +2793,25 @@
...
+                        defined in [[!RFC3986]] and obtain the
+                        <var>scheme name</var>. If the parsing based
+                        on the syntax defined in [[!RFC3986]] fails,
+                        <a>throw</a> a <code>SyntaxError</code>.  If
+                        the <var>scheme name</var> is not implemented
+                        by the browser <a>throw</a> a
+                        <code>NotSupportedError</code>.  If
+                        <var>scheme name</var> is <code>turn</code> or
+                        <code>turns</code>, and parsig the
```
Comment (fippo) [inline webrtc.html]: parsig -> parsing

File: webrtc.html [hunk 3]
```diff
@@ -2793,12 +2793,25 @@
...
+                        <var>url</var> using the syntax defined in
+                        [[!RFC7064]] fails, <a>throw</a> a
+                        <code>SyntaxError</code>.  If <var>scheme
```
Comment (fippo) [inline webrtc.html]: double whitespace

File: webrtc.html [hunk 4]
```diff
@@ -2793,12 +2793,25 @@
...
+                        name</var> is <code>stun</code> or
+                        <code>stuns</code>, and parsig the
```
Comment (fippo) [inline webrtc.html]: parsig -> parsing

File: webrtc.html (hunk 3 above)
Comment (fippo) [inline webrtc.html]: Also on line 2804 but I missed that

File: webrtc.html (hunk 4 above)
Comment (aboba) [inline webrtc.html]: Fixed spelling and whitespace issues.

**Comment by stefhak** [comment]
LGTM.

**Pull request merged**